agent/pkg/utils/truncating_logger.go

```diff
@@ -24,21 +25,21 @@ func (truncatingLogger *TruncatingLogger) Info(_ context.Context, message string
 	if truncatingLogger.LogLevel < logger.Info {
 		return
 	}
-	rlog.Errorf("gorm info: %.150s", message)
+	loggerShared.Log.Errorf("gorm info: %.150s", message)
```

Comment:
do we need this file after the log refactor? is it used?

Reply:
hmm as the main author I would like to address this question to @RamiBerm :)
feels like a hot potato game

Reply:
This is still needed, its used to override the default sqlite logger.
The sqlite logger tends to log entire SQL queries for warnings/errors which can be huge for large entries, this class ensures the log lines are truncated at a certain point